jupyter_lidar_inspection.ipynb

```diff
@@ -0,0 +1,918 @@
+    "series.add_rasters(rasters_different_res_hs)\n",
+    "series.show()\n",
+    "# You can see here that better resolution doesn't mean more detail."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ccb7f8b",
+   "metadata": {},
+   "source": [
+    "Is that composite better or worse than an interpolated, raw one for your future data analysis?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9bfce78e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "gs.run_command('r.resamp.bspline',\n",
+    "               input='point_cloud_100', # I decided that this one has a good resolution:detail payoff\n",
+    "               output='point_cloud_100_intp',\n",
+    "               method='bicubic')\n",
+    "gs.run_command(\"r.relief\",input='point_cloud_100_intp',output='point_cloud_100_intp_hs',altitude=30,azimuth=315)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08833664",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "series = gj.SeriesMap(height = 700)\n",
+    "series.add_rasters(['dtm_patched_hs','point_cloud_100_intp_hs'])\n",
+    "series.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6693fdfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = gj.InteractiveMap(width=700, height=700, tiles=None)\n",
+    "m.add_raster('dtm_patched_hs')\n",
+    "m.add_raster('point_cloud_100_intp_hs')\n",
+    "m.add_layer_control()\n",
+    "m.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e2b55e8",
+   "metadata": {},
+   "source": [
+    "Let's first interpolate the missing values in the patched DTM and then dig a bit more."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97706b1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "gs.run_command('r.resamp.bspline',               \n",
+    "               input='dtm_patched',\n",
+    "               output='dtm_patched_int',\n",
+    "               method='bicubic')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24cdeade",
+   "metadata": {},
+   "source": [
+    "Guess, what: GRASS has also a ton of addons and we can install them easily also here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0d9ea7d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can see differences on microtopography:\n",
+    "# Let's denoise and see how does local noise affect the elevation mode\n",
+    "\n",
+    "# Let's install the extension\n",
+    "gs.run_command(\"g.extension\",\n",
+    "               extension='r.denoise',\n",
+    "               operation='add')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd959956",
+   "metadata": {},
+   "source": [
+    "Create a temporary directory or if you created one in the beginning just use that."
```

Comment:
Please explain why you do these steps, and maybe better just use here the same temp directory you created to avoid confusion, i.e., keep it simple